android/TodokTodok/app/src/main/java/com/team/todoktodok/presentation/view/discussion/create/vm/CreateDiscussionRoomViewModel.kt

```
@@ -116,18 +187,18 @@ class CreateDiscussionRoomViewModel(
                 )
             }
         } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)